app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
```

Comment:
The import statement should use absolute imports instead of relative imports. Change `from app.errors import NotWearingMaskError, VaccineError` to `from errors import NotWearingMaskError, VaccineError` to comply with the checklist.